.github/workflows/test_frontend_e2e_live_dev.yml

```diff
@@ -1,7 +1,8 @@
-# TODO: 1. run local: (i) start backend (a) local backend in GH action, or (b) use dev's backend URL (won't always work), (ii) set to on:pr&push
-# TODO: 2. after '1': expose params to workflow_dispatch: so can choose which env we want to test, in case we don't want to run all 3 at once
+# Test Frontend deployment using Playwright End-to-End tests.
+# todo: 1. run local: (i) start backend (a) local backend in GH action, or (b) use dev's backend URL (won't always work), (ii) set to on:pr&push
+# todo: 2. after '1': expose params to workflow_dispatch: so can choose which env we want to test, in case we don't want to run all 3 at once
 
-name: Test - Frontend deployments - End to end
+name: Test, Frontend Dev, Playwright E2E
```

Comment:
## Dev
- Add: Develop GH action that tests on dev using the code / playwright tests that were written at the time dev was deployed.